client/src/components/Form/FormDisplay.vue

```diff
@@ -146,6 +145,9 @@ export default {
             this.onChange(true);
         },
         onCloneInputs() {
+            visitInputs(this.inputs, (input) => {
+                input.error = null;
```

Comment:
Hard to tell from here, are you sure you're not mutating the inputs prop here ?

Reply:
You are right, we can avoid adding the `error` attribute upfront to the each input object in the incoming inputs array and instead add them to its local clone using `Vue.set`. Thank you.